doc/dracula.txt

```diff
@@ -56,19 +56,19 @@ In the following section, `1` signifies `on` and `0` signifies `off`.
 * *g:dracula_bold*
 Include bold attributes in highlighting >
     let g:dracula_bold = 1
-<
+
```

Comment:
Not sure these are necessary but /shrug

Reply:
`<` is only necessary if you want to end the code block on a line that doesn't begin on column 1. Removed just because they were confusing when i went in and added the new option.

Reply:
Got it.